Sources/xcodeproj/Errors/Errors.swift

```diff
@@ -21,6 +22,8 @@ public enum XCodeProjError: Error, CustomStringConvertible {
             return "The project doesn't contain a .pbxproj file at path: \(path.string)"
         case let .xcworkspaceNotFound(path):
             return "The project doesn't contain a .xcworkspace at path: \(path.string)"
+        case let .pbxprojIncorrectSerializationType(path, expectedType, serialized):
+            return "Can't serialize pbxproj plist at path: \(path.string) to: \(expectedType) but serialized result type is \(type(of: serialized)): \(serialized) "
```

Comment:
Nitpick: I'd rephrase this one to:

```
Can't serialise \(serialized) into a \(expectedType) for project at path \(path)
```

Reply:
Change throw on force cast and remove this error